conf.py

```diff
@@ -185,7 +185,10 @@
 def setup(app):
 # add_stylesheet() was renamed to add_css_file() in sphinx 1.8 released
 # in September 2018. add_stylesheet() will be removed in sphinx 4.0
-   app.add_stylesheet("sof-custom.css")
+    try:
+        app.add_css_file('sof-custom.css')
+    except:
```

Comment:
Can you please catch a more specific exception? It should be printed when it fails.

Reply:
@marc-hb, sure, let me take a look. But I don't think there is a point to print anything.
https://thesofproject.github.io/latest/contribute/process/docbuild.html states that we need Sphinx 1.7.5 or newer. add_css_file() is added with 1.8.0, it is not an error if we need to fall back to add_stylesheet.

off topic: Do we want to fix the other existing warnings from pylint?

Reply:
@marc-hb, pushed a new version to catch the AttributeError exception which will be raised when the add_stylesheet is not available (pre Sphinx 1.8.0).

Reply:
> But I don't think there is a point to print anything.

I don't think that either, this was just to help you find `AttributeError`.

> off topic: Do we want to fix the other existing warnings from pylint?

Maybe some of them in a different PR why not.